game-core/src/main/java/games/strategy/triplea/ai/pro/ProTechAi.java

```diff
@@ -200,12 +200,6 @@ private static float getStrengthOfPotentialAttackers(final Territory location, f
             int availOther = 0;
             for (final Unit candidateTransport : transports) {
               final Collection<Unit> thisTransUnits = TransportTracker.transporting(candidateTransport);
```

Comment:
Would you mind changing the Javadocs for `TransportTracker#transporting(Unit)` to no longer indicate the return value  may be `null`?